README.md

```diff
@@ -189,16 +190,18 @@ func handler(w http.ResponseWriter, r *http.Request) {
 
 func main() {
     r := mux.NewRouter()
-    r.HandleFunc("/", handler)
-    r.HandleFunc("/products", handler)
+    r.Methods("GET").HandleFunc("/", handler)
```

Comment:
Don't modify these - should show both the simple (non-method) and precise (provide a method) use-cases.